<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>98</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AtmoSense</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="centralLayout">
    <item>
     <layout class="QHBoxLayout" name="headerHLayout">
      <item>
       <widget class="QLabel" name="locationLabel">
        <property name="styleSheet">
         <string notr="true">font-size:20px;font-weight:600;padding:6px;</string>
        </property>
        <property name="text">
         <string>Select location</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="settingsButton">
        <property name="text">
         <string>Settings</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="recentButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/icons/clock.png</normaloff>:/icons/clock.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="changeLocationButton">
        <property name="text">
         <string>Change</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="detectLocationButton">
        <property name="text">
         <string>üìç Detect</string>
        </property>
        <property name="toolTip">
         <string>Detect current location via IP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="refreshButton">
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="WeatherSummaryWidget" name="weatherSummaryWidget" native="true"/>
    </item>
    <item>
     <widget class="HourlyForecastWidget" name="hourlyForecastWidget" native="true"/>
    </item>
    <item>
     <widget class="DailyForecastWidget" name="dailyForecastWidget" native="true"/>
    </item>
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="styleSheet">
       <string notr="true">
        font-size: 13px;
        color: #666;
        padding: 4px;
       </string>
      </property>
      <property name="text">
       <string>‚è≥ Waiting for update...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WeatherSummaryWidget</class>
   <extends>QWidget</extends>
   <header>weathersummarywidget.h</header>
  </customwidget>
  <customwidget>
   <class>HourlyForecastWidget</class>
   <extends>QWidget</extends>
   <header>hourlyforecastwidget.h</header>
  </customwidget>
  <customwidget>
   <class>DailyForecastWidget</class>
   <extends>QWidget</extends>
   <header>dailyforecastwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>